test-suite/Makefile

```
@@ -260,6 +253,22 @@ summary.log:
 report: summary.log
 	$(HIDE)bash report.sh
 
+#######################################################################
+# Test suite configuration, to eventually go away
+#######################################################################
+
+# Not inside Dune, build by hand
+ifeq ($(INSIDE_DUNE),)
+# EGJA: using $(ROOT)/_build/default/... here confuses make due to quotes...
+TESTCONFIGURE:=../_build/default/test-suite/tools/coq_config_to_make.exe
+$(TESTCONFIGURE): tools/coq_config_to_make.ml
+	dune build ./tools/coq_config_to_make.exe
+
+$(TEST_SUITE_CONFIG): $(TESTCONFIGURE)
+	chmod u+w $@ || true # Be more user-friendly w.r.t. Dune cache read-only stuff
+	$(TESTCONFIGURE) > $@
```

Would 
```suggestion
$(TEST_SUITE_CONFIG): tools/coq_config_to_make.ml
	dune build $@
```
work here? then we don't fight the dune cache

---

ping @ejgallego

---

ping @ejgallego

---

Had to tweak the variables slightly, as the command has to be the basename, but when called directly from the makefile we need to preprend the `_build` path, but indeed, that seems like a good idea, and seems to work too, thanks.